client/profile-wizard/index.js

```diff
@@ -278,11 +258,29 @@ class ProfileWizard extends Component {
 			} );
 	}
 
+	checkPluginsAreInstalled() {
+		const {
+			activePlugins,
+			profileItems,
+			updateProfileItems,
+			profileItemsLoaded,
+		} = this.props;
+
+		// Track plugins if already installed.
+		if (
+			profileItemsLoaded &&
```

Comment:
Could you add a `isUpdatingProfileItems` check here as well, or have the `updateProfileItems` update optimistically.
I encountered an "almost" infinite loop of triggering a profile items update (see Gif), it did eventually stop.
To reproduce, I went through your initial testing instructions, and then later through the wizard again adding **Woocommerce Shipping** this time, so it would hit the `if`statement below.
You can also reproduce this on a fresh install and selecting both jetpack and woo shipping (woo services)
![plugin-installed](https://user-images.githubusercontent.com/2240960/110345007-7ec50e00-8004-11eb-8acf-b9c51fa5f6f3.gif)

Reply:
That makes sense this would happen, thanks for catching this 🤔 I think probably checking that updating is in progress makes the most sense.

Reply:
Talked to some other team members, I think we can remove `plugins` altogether. I'll update the PR and description accordingly.